shared/src/main/java/uk/gov/di/authentication/entity/TICFCRIRequest.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.di.authentication.entity;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+import uk.gov.di.authentication.shared.validation.Required;
+
+import java.util.List;
+
+public record TICFCRIRequest(
+        @Expose @Required @SerializedName("sub") String internalPairwiseId,
```

Comment:
Consider calling the variable sub, then the @SerializedName annotation can be removed.

Reply:
I'm going to keep this one as internalPairwiseId since sub is a bit obscure within our domain, but agree on all the others 👍